class2.1/task3.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <string>
+//I create history.txt file for saving data.
+using namespace std;
+
+map<string, int> userHistory;
+
+void loadHistory() {
+    ifstream file("history.txt");
+    string name;
+    int count;
+    while (file >> name >> count) {
+        userHistory[name] = count;
+    }
+    file.close();
+}
+
+void saveHistory() {
+    ofstream file("history.txt");
+    for (const auto& entry : userHistory) {
+        file << entry.first << " " << entry.second << endl;
```

Comment:
Try something like this. std::println can work with any stream stdcout, stderr or any ostream object 
```
for (const auto& [name, count] : userHistory) {
      std::println(file, std::format("{} {}", name, count));
}
```